sdks/go/pkg/beam/core/runtime/graphx/serialize.go

```diff
@@ -207,45 +207,45 @@ func encodeFn(u *graph.Fn) (*v1.Fn, error) {
 	case u.Fn != nil:
 		fn, err := encodeUserFn(u.Fn)
 		if err != nil {
-			return nil, fmt.Errorf("bad userfn: %v", err)
+			return nil, fmt.Errorf("failed to encode function %v, bad userfn: %v", u, err)
```

Comment:
("failed to encode DoFn %v, bad userfn: %v", u, err)

Reply:
Done